static/app/utils/domId.tsx

```diff
@@ -2,4 +2,4 @@ function domId(prefix: string): string {
   return prefix + Math.random().toString(36).substr(2, 10);
 }
 
-export {domId};
+export default domId;
```

Comment:
isn't named export better?

Reply:
I think we usually use default export if it's just a single util